src/Integration/Integration.csproj

```diff
@@ -25,7 +25,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
+    <OutputPath>bin\Debug\VS$(VersionSpecificSuffix)\</OutputPath>
```

Comment:
Is this project version specific?

Reply:
Yes; it imports the legacy VS ruleset assembly.